pkg/gui/controllers/local_commits_controller.go

```diff
@@ -452,6 +452,10 @@ func (self *LocalCommitsController) amendTo(commit *models.Commit) error {
 		return self.c.Refresh(types.RefreshOptions{Mode: types.ASYNC})
 	}
 
+	if self.git.Status.WorkingTreeState() != enums.REBASE_MODE_NONE {
+		return self.c.ErrorMsg(self.c.Tr.AlreadyRebasing)
+	}
```

Comment:
We are using the same error message ("Can't perform this action during a rebase") both for non-head real commits and for todo commits. I think this is good enough and it's not worth adding a better (more specific) text for todo commits; but we could if you disagree.